CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 ### Removed
 
+## [6.0.2] 2021-07-05
+### Changed
+- `List.from_to_step` now forbids non-positive step
+- Fixed leaking state between contracts
```

Comment:
That sounds a bit worrying, but hopefully those reading it realize this is the compiler and this bug doesn't affect on-chain contracts 😅

Reply:
aeh, mentioned

Reply:
Was more thinking about "leaking state between contracts" but fair enough 😅